py/mkrules.mk

```diff
@@ -105,7 +105,7 @@ endif
 ifneq ($(FROZEN_MPY_DIR),)
 # to build the MicroPython cross compiler
 $(TOP)/mpy-cross/mpy-cross: $(TOP)/py/*.[ch] $(TOP)/mpy-cross/*.[ch] $(TOP)/ports/windows/fmode.c
-	$(Q)$(MAKE) -C $(TOP)/mpy-cross
+	$(Q)$(MAKE) -C $(TOP)/mpy-cross MODULE_DIR=
```

Comment:
Note: this is required to get `mpy-cross` to work. The cross compiler won't need these custom modules anyway.

Reply:
I think it's still necessary, in case your modules use features that are not enabled (or different) in the cross compiler.